civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -155,5 +160,13 @@ function filterTable(statusId) {
     $filtersDropdown.on('change', function (e) {
       filterTable($(this).val());
     });
+
+    // Listen for ready event when T&A finishes loading all modules
+    document.addEventListener('taReady', function (e) {
```

Comment:
who is sending the `taReady` event in this case? What's the initialization flow here (= sequence of events)?